scr/main/java/bittercode/util/StoreUtil.java

```diff
@@ -0,0 +1,86 @@
+package com.bittercode.util;
+
+import java.io.PrintWriter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import com.bittercode.model.UserRole;
+
+/*
+ * Store UTil File To Store Commonly used methods
+ */
+public class StoreUtil {
+
+    /**
+     * Check if the User is logged in with the requested role
+     */
+    public static boolean isLoggedIn(UserRole role, HttpSession session) {
+
+        return session.getAttribute(role.toString()) != null;
+    }
+
+    /**
+     * Modify the active tab in the page menu bar
+     */
+    public static void setActiveTab(PrintWriter pw, String activeTab) {
+
+        pw.println("<script>document.getElementById(activeTab).classList.remove(\"active\");activeTab=" + activeTab
+                + "</script>");
+        pw.println("<script>document.getElementById('" + activeTab + "').classList.add(\"active\");</script>");
+
+    }
+
+    /**
+     * Add/Remove/Update Item in the cart using the session
+     */
+    public static void updateCartItems(HttpServletRequest req) {
+        String selectedBookId = req.getParameter("selectedBookId");
+        HttpSession session = req.getSession();
+        if (selectedBookId != null) { // add item to the cart
+
+            // Items will contain comma separated bookIds that needs to be added in the cart
+            String items = (String) session.getAttribute("items");
+            if (req.getParameter("addToCart") != null) { // add to cart
+                if (items = null || items.length() = 0)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Possible comparison operator misuse</b></div>
<div id="fix">

It appears that there might be a comparison issue in the conditions. Consider using `==` for equality comparison instead of `=` which is used for assignment. A similar issue was also found in scr/main/java/bittercode/util/StoreUtil.java (line 78-81).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -45,1 +45,1 @@
 -                if (items = null || items.length() = 0)
 +                if (items == null || items.length() == 0)
 @@ -68,1 +68,1 @@
 -                if (itemQty = 1) {
 +                if (itemQty == 1) {
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434572244>#7fa0a4</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly